lib/ui/screens/group_screen.dart

```diff
@@ -0,0 +1,307 @@
+import 'package:flutter/material.dart';
```

Comment:
This entire file should really be split up, one for each screen

Reply:
Pretty sure this is done the same in the other screens

Reply:
I couldn't find any cases of this, and it should be split regardless of how others do it

Reply:
I think keeping these screens in 1 place is fine, because it is effectively a single screen. There is no visual difference, the splitting is purely necessary for the API (boards have years in their url and not a pk).

Reply:
Ah ok, I didn't look at the GUI, it if its the same screen it might be possible to combine them using generics or smth but that seems like a lot of work